packages/launchpad/cypress/e2e/base-url.cy.ts

```diff
@@ -0,0 +1,17 @@
+describe('baseUrl', () => {
```

Comment:
Nit: can we add some context describe how this is specific to the e2e testing type?